src/main/webapp/js/finscholar/scholarshippageview/scholarshippageview.soy

```diff
@@ -0,0 +1,109 @@
+{namespace example.templates.scholarshippageviews}
+{template .scholarshippage}
+  {@param scholarship: [generalInfo: [scholarshipName: string,  
+                                       schoolsList:string,
+                                       introduction: string,
+                                       URL: uri],
+                        requirements: [academicRequirements: string,
+                                       ethnicityRaceRequirements: string,
+                                       financialRequirements: string,
+                                       genderRequirements: string,
+                                       locationRequirements: string,
+                                       nationalOriginRequirements: string,
+                                       otherRequirements: string],
+                        applicationNotes: [isRenewable: bool,
+                                           amountPerYear: string,
+                                           applicationProcess: string,
+                                           numberOfYears: int]
+                        ]}
+  <section class="section--center mdl-grid mdl-grid--no-spacing mdl-shadow--2dp">
+    <div class="mdl-card__supporting-text mdl-grid mdl-grid--no-spacing">
+      <div class="page-content">
+        {call .scholarshipIntroduction}
+          {param generalInfo: $scholarship.generalInfo /}
+        {/call}
+        {call .scholarshipRequirements}
+          {param requirements: $scholarship.requirements /}
+        {/call}
+        {call .scholarshipApplication}
+          {param applicationNotes: $scholarship.applicationNotes /}
+        {/call}
+      </div>
+    </div>
+  </section>
+{/template}
+
+{template .scholarshipIntroduction}
+  {@param generalInfo: [scholarshipName: string,  
+                         schoolsList: string,
+                         introduction: string,
+                         URL: uri]}
+  <div class="mdl-card mdl-cell mdl-cell--12-col">
+    <div id="scholarship-header">
+      <h4>{$generalInfo.scholarshipName}</h4>
```

Comment:
Just curious, but is there a reason why you're starting with heading level 4 (`<h4>`)? I'm asking because if it's for aesthetics, we should figure out how to make it more centered with the UI library because this confuses a screen reader that expects the headings to be in a hierarchy.

Reply:
I just changed to a more reasonable styling template. Thanks for the reminder!